pkg/injector/pod_patch.go

```diff
@@ -142,10 +146,57 @@ func (i *injector) getPodPatchOperations(ar *v1beta1.AdmissionReview,
 			Value: value,
 		},
 	)
+	patchOps = append(patchOps, envPatchOps...)
 
 	return patchOps, nil
 }
 
+func setDaprEnvVars(list []corev1.Container, basePath string) []PatchOperation {
+	envPatchOps := []PatchOperation{}
+	portEnv := []corev1.EnvVar{
+		{
+			Name:  userContainerDaprHTTPPortName,
+			Value: strconv.Itoa(sidecarHTTPPort),
+		},
+		{
+			Name:  userContainerDaprGRPCPortName,
+			Value: strconv.Itoa(sidecarAPIGRPCPort),
+		},
+	}
+	var value interface{}
```

Comment:
why do we need to define var here? this var is only used in the scope of line 181.